src/util/saveArticles.ts

```diff
@@ -244,12 +260,13 @@ export async function saveArticles(zimCreator: ZimCreator, downloader: Downloade
   const cssModuleDependencies = new Set<string>()
   let jsConfigVars = ''
   let prevPercentProgress: string
-
+  const desktopRenderer = new RendererBuilder('desktop')
```

Comment:
Here is should clearly be `mainpage_renderer` and `article_renderer`

Reply:
Done.There are `mainPageRenderer` and `articlesRenderer` now.